src/components/header/index.js

```diff
@@ -87,25 +88,31 @@ const Hamburger = ({ open, ...props }) => (
 // nested nav renderer
 const Nav = ({ routes, current, ...props }) => (
 	<nav {...props}>
-		{routes.map(route => (
-			<NavItem
-				to={route}
-				current={current}
-				data-route={getRouteIdent(route)}
-				class={cx(
-					route.class,
-					(pathMatchesRoute(current, route) ||
-						(route.content === 'guide' && /^\/guide\//.test(current)) ||
-						(route.content === 'blog' && /^\/blog\//.test(current))) &&
-						style.current
-				)}
-			/>
-		))}
+		{routes.map(route =>
+			route.routes ? (
+				<NavMenu
+					to={route}
+					current={current}
+					data-route={getRouteIdent(route)}
+				/>
+			) : (
+				<NavLink
+					to={route}
+					class={cx(
+						route.class,
+						(pathMatchesRoute(current, route) ||
+							(route.content === 'guide' && /^\/guide\//.test(current)) ||
+							(route.content === 'blog' && /^\/blog\//.test(current))) &&
+							style.current
+					)}
+				/>
+			)
+		)}
```

Comment:
I thought it made a bit more sense to switch `NavItem` to `NavMenu`, and avoid passing `NavLinks` directly through.